lib/src/firebase_performance_dio.dart

```diff
@@ -16,12 +16,14 @@ class DioFirebasePerformanceInterceptor extends Interceptor {
   DioFirebasePerformanceInterceptor({
     this.requestContentLengthMethod = defaultRequestContentLength,
     this.responseContentLengthMethod = defaultResponseContentLength,
+    this.generateKey,
   });
 
   /// key: requestKey hash code, value: ongoing metric
-  final _map = <int, HttpMetric>{};
+  final _map = <dynamic, HttpMetric>{};
```

Comment:
Can we use a specific type instead of `dynamic` ?

Reply:
I was looking for a way to store int (for the default implementation) and other types, like String (from user defined method "generateKey") values.